src/packages/client/src/__tests__/integration/happy/exhaustive-schema/__snapshots__/test.ts.snap

```diff
@@ -1315,15 +1315,45 @@ export namespace Prisma {
   }
 
   export type AggregatePostArgs = {
+    /**
+     * Filter which Post to aggregate.
+    **/
     where?: PostWhereInput
+    /**
+     * Determine the order of Posts to aggregate.
+    **/
     orderBy?: Enumerable<PostOrderByInput>
+    /**
+     * Sets the start position
+    **/
     cursor?: PostWhereUniqueInput
+    /**
+     * The number of Posts to aggregate. If negative number, it will take Posts before the \`cursor\`.
```

Comment:
It will actually take the cursor too in results without `skip: 1`

So we need to find a better wording because "before" makes me think the cursor is not included but it is and you need to pass `skip: 1` to exclude it

Example from docs
```
Example: Paging backwards with cusor-based pagination

To page backwards, set take to a negative value. The following query returns 4 Post records with an id of less than 200, excluding the cursor:

const myOldCursor = 200
const firstQueryResults = prisma.post.findMany({
  take: -4,
  skip: 1,
  cursor: {
    id: myOldCursor,
  },
  where: {
    title: {
      contains: 'Prisma' /* Optional filter */,
    },
  },
})
```

Reply:
Takes `n` Posts from the position of the cursor, including the cursor?

Reply:
`Takes n Posts from the position of the cursor, cursor included.` maybe

Reply:
I have changed it to 
```
Take \`n\` Posts from the position of the cursor (cursor included). If negative number, it will take Posts before the \`cursor\`.
```

Reply:
So the cursor is always included from what I understand.
This suggests the cursor is included only in one case